openjdk/src/test/java/org/conscrypt/OpenSSLX509CertificateTest.java

```diff
@@ -43,10 +45,28 @@ public void testSerialization_NoContextDeserialization() throws Exception {
 
             // Mark the field as non-final on JVM that need it.
             try {
-                Field modifiersField = Field.class.getDeclaredField("modifiers");
-                modifiersField.setAccessible(true);
-                modifiersField.setInt(targetUID, targetUID.getModifiers() & ~Modifier.FINAL);
-            } catch (NoSuchFieldException ignored) {
+                Field modifiersField = null;
+                try {
+                    Field.class.getDeclaredField("modifiers");
```

Comment:
`modifiersField = Field.class.getDeclaredField("modifiers");` ?

Reply:
doh, good call.. only ran this on a newer OpenJDK